apps/web/src/app/[lang]/page.tsx

```diff
@@ -13,20 +14,25 @@ export default async function Index({ params: { lang } }: PageProps) {
   const daos = await getDaoList({ token: newtoken.accessJWT });
 
   return (
-    <div data-testid="dho-list-container" className="grid grid-cols-4 gap-4 px-6 py-4">
-      {daos.map((dao) => (
-        <div key={dao.name}>
-          <Link href={getAssignmentsPath(lang, dao.url as string)}>
-            <CardOrganisation
-              createdDate={dao.date}
-              description={dao.description as string}
-              icon={dao.logo}
-              members={0}
-              title={dao.title as string}
-            />
-          </Link>
+    <div className="w-full px-6 py-4 overflow-auto">
+      <Container>
+        <div data-testid="dho-list-container" className="w-full">
+          {daos.map((dao) => (
+            <div key={dao.name} className="mb-5">
+              <Link href={getAssignmentsPath(lang, dao.url as string)}>
+                <CardOrganisation
+                  createdDate={dao.date}
+                  description={dao.description as string}
+                  icon={dao.logo}
+                  members={0}
+                  agreements={0}
```

Comment:
there is a type error because of missing type `agreements` on `CardOrganisation`